packages/x-date-pickers/README.md

```diff
@@ -15,12 +15,14 @@ npm install @mui/x-date-pickers
 yarn add @mui/x-date-pickers
 ```
 
-This component has three peer dependencies that you will need to install as well.
+This component has four peer dependencies that you will need to install as well.
 
 ```json
 "peerDependencies": {
-  "@mui/material": "^5.0.0",
-  "react": "^17.0.0"
+  "@mui/material": "^5.2.3",
+  "@mui/system": "^5.2.3",
+  "react": "^17.0.2 || ^18.0.0",
+  "react-dom": "^17.0.2 || ^18.0.0"
 },
 ```
 
```

Comment:
Change link to documentation below to https://mui.com/x/react-date-pickers/getting-started/ ?
Same with `@mui/x-date-pickers-pro` readme